.github/workflows/ci.yml

```diff
@@ -161,7 +161,12 @@ jobs:
         platform:
           [
             {
-              os: "macos-latest",
+              os: "macos-14",  # first available arm macos runner
+              python-architecture: "x64",
+              rust-target: "aarch64-apple-darwin",
```

Comment:
Why "x64" and "aarch64"?

Reply:
An oversight from me that probably needs correcting 👀